app/src/common/components/MovieList/MovieElement/Poster/Poster.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Poster.scss';
+
+export default class Poster extends React.Component {
```

Comment:
Having state is not required. I would implement it in this way:
```jsx
const Poster = ({ src = 'http://via.placeholder.com/500x750', title = 'No title provided' }) => (
  <img styleName="poster" src={src} alt={`Poster of ${title}`} />
);
```

Reply:
done